Makefile

```diff
@@ -239,4 +239,10 @@ run-performance-tests:
 list-nunit-tests:
 	$(MSBUILD) $(MSBUILD_FLAGS) $(TEST_TARGETS) /t:ListNUnitTests
 
+import-docs:
+	cp "$(call GetPath,MicrosoftDocs)/docs/android/deploy-test/building-apps/build-process.md" Documentation/guides/BuildProcess.md
```

Comment:
As per our 1:1, we shouldn't need the `$(MicrosoftDocsFullPath)` MSBuild property, we can instead use a `$(XAMARIN_DOCS_PATH)` make variable (naming debatable).

Additionally, this should sync the `Documentation/guides/messages` directory.